workers/unity/Assets/Fps/Scripts/SetupLogic/AndroidWorkerConnector.cs

```diff
@@ -0,0 +1,183 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.GameObjectCreation;
+using Improbable.Gdk.GameObjectRepresentation;
+using Improbable.Gdk.Mobile;
+using Improbable.Gdk.Mobile.Android;
```

Comment:
```suggestion
#if UNITY_ANDROID
using Improbable.Gdk.Mobile.Android;
#endif
```

Reply:
My suggestions in this file will fix building of non-android workers (e.g. GameLogicWorker)